pkg/sql/codegen/optimize/template_optimize.go

```diff
@@ -57,16 +57,20 @@ class CustomOptFlowRunner(BaseOptFlowRunner):
 
 const optFlowSubmitText = `
 import os
+import json
```

Comment:
It doesn't make sense to have a template this long -- please file a PR to minimize the template into only one function call which takes all template-filler parameters, before merging this one.

Reply:
Done. Move most Python code to `runner.py`.